src/PresentationalComponents/Truncate/Truncate.js

```diff
@@ -24,6 +24,10 @@ class Truncate extends React.Component {
         });
     }
 
+    dangerousHtml(text) {
+        return { __html: text };
```

Comment:
Do we want to sanitize this html before setting it?

If so, we may want to use something like: https://github.com/cure53/DOMPurify

Reply:
Sounds like a good idea. I'll look into it. Thanks for the review!